bin/birdhouse

```diff
@@ -561,6 +561,18 @@ parse_backup_create_args() {
       parse_backup_create_args "$@"
       ;;
     '')
+      if
+        [ "${BIRDHOUSE_BACKUP_NO_RESTIC}" = 'true' ] && \
+        [ -z "${BIRDHOUSE_BACKUP_VOLUME}" ]; then
+        source_env 2> /dev/null
```

Comment:
Depending on the log settings this could write to a file or stdout so redirecting stderr to /dev/null won't silence the message.

Also, if the goal is simply to get access to the `log` command we don't need to source the entire env. We could just do:

```sh
eval "$(print_log_command)"
```

If we want to get fancy we could create a `source_log_env` function that does the same thing as `source_env` but just for the log functions but that might be overkill since the log script runs much much quicker than loading the environment.

**BUT** since I recommend we move this to the `do_backup_create_restore` command anyway we should just do that for now and then the log function is already available there.